### DISCUSSION TITLE
Control of the brightness via the keyboard stopped working for the external display after app update

### DISCUSSION THREAD

**Discussion by tropxy** [MonitorControl/MonitorControl#1677] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Discussion

I am writing this down so that if this happens with someone else, that person can find the solution as I did.

Laptop: MacBook Pro 2021, M1 Pro
OS: Mac OS Sequoia 15.1 (24B83)

After update to the recent version (4.3.3) the control of the brightness via the keyboard stopped working for the external display, I have unplugged and plugged again the display which is connected via USB type C, but nothing changed. 

The solution was to end the MonitorControl application and restart it, which caused the app to verify that not full control was granted to the "accessibility features". Thus after granting accessibility to the app, the functionality started working again

**Comment by razvanbh** [comment]
I had the same issue but with controlling the volume via keyboard on a fresh install.
I made it work by allowing the accessibility features from system settings and restarting the app.
